getting-started/iterate-new-app-intellij.hbs.md

```diff
@@ -0,0 +1,141 @@
+5. View the changes to your workload running on the cluster.
+6. Either continue making changes, or stop and deactivate the live update when finished. To stop Live Update navigate to the `Run` tab at the bottom left of the IntelliJ window and click the red stop icon on the left side of the screen.
+
+## <a id="debug-your-app"></a>Debug your application
+
+Debug the cluster either on the application or in your local environment.
+
+Use the following steps to debug the cluster:
+
+1. Set a breakpoint in your code.
+1. Create a Run Configuration.
+   1. In IntelliJ, select the `Edit Run/Debug configurations` dialog in the top-right corner. Alternatively, navigate to `Run` > `Edit Configurations`.
+   1. Select `Tanzu Debug Workload`.
+   1. Select `Add new run configuration`, or click the plus icon at the top of the list.
+   1. Give your new run configuration a name, for example `Tanzu Debug Workload - tanzu-java-web-app`.
+   1. In the **Workload File Path** field, provide the path to the `workload.yaml` file in the Tanzu Java Web App project directory located at `Config` > `workload.yaml`. Select the folder icon on the right-side of the field, navigate to the Tanzu Java Web App directory, select the `workload.yaml` file and click the `Open` button. The `workload.yaml` provides configuration instructions about your application to the Tanzu Application Platform.
+   1. In the **Local Path** field, provide the path to the directory containing the Tanzu Java Web App. The Local Path value tells the Tanzu Developer Tools extension which directory on your local file system to bring into the source image container image.
+   For example, `/Users/developer/Documents/tanzu-java-web-app`.
+   1. In the **Source Image** field, provide the destination image repository to publish an image containing your workload source code. The Source Image value tells the Tanzu Developer Tools extension where to publish the container image with your uncompiled source code, and what to name that image. The image must be published to a container registry where you have write (push) access.
+   For example, `gcr.io/myteam/tanzu-java-web-app-source`.
+
+   >**Note** Consult the documentation for the registry you're using to determine which steps are necessary to authenticate and gain push access.
+
+   >For example, if you use docker consult [docker's docs](https://docs.docker.com/engine/reference/commandline/login/), if you use Harbor consult [Harbor's docs](https://goharbor.io/docs/1.10/working-with-projects/working-with-images/pulling-pushing-images/), etc.
+
+   8. In the **Namespace** field, optionally provide the namespace that the workload should be associated with on the cluster. If you followed the steps to [Prepare your IDE to iterate on your application](/iterate-new-app-intellij.hbs.md#prepare-to-iterate) you do not need to enter a namespace and IntelliJ will use the namespace you associated with your context.
```

Comment:
Apply the changes?

Reply:
good catch, looks like i missed this in all of our run configs